praw/models/listing/mixins/redditor.py

```diff
@@ -1,17 +1,21 @@
 """Provide the RedditorListingMixin class."""
+from typing import Any, Dict, Generator, TypeVar, Union
 from urllib.parse import urljoin
 
 from ....util.cache import cachedproperty
 from ..generator import ListingGenerator
 from .base import BaseListingMixin
 from .gilded import GildedListingMixin
 
+Reddit = TypeVar("Reddit")
+_SubListing = TypeVar("_SubListing")
```

Comment:
Ordering or _.

Reply:
Understood

Reply:
While it breaks the class sort order, I think it'd be preferable to define the `SubListing` class before `RedditorListingMixin` to avoid needing to use an underscore. Will you do that please?

Reply:
Sure.